packages/terra-functional-testing/tests/jest/commands/utils/describeViewports.test.js

```diff
@@ -0,0 +1,23 @@
+const describeViewports = require('../../../../src/commands/utils/describeViewports');
+
+describe('describeViewports', () => {
+  it('should describe viewport', () => {
+    const mockBefore = jest.fn();
+    const mockDescribe = jest.fn();
+    const mockAfter = jest.fn();
+
+    global.before = mockBefore;
+    global.describe = mockDescribe;
+    global.after = mockAfter;
+
+    global.Terra = {
+      serviceOptions: {
+        formFactor: 'tiny',
+      },
+    };
+
+    describeViewports('viewport', ['tiny', 'large'], () => {});
+
+    expect(global.describe).toHaveBeenCalled();
```

Comment:
```suggestion
    expect(global.describe).toHaveBeenCalledOnce();
```

Reply:
This doesn't appear to be a supported expect assertion

```
 FAIL  packages/terra-functional-testing/tests/jest/commands/utils/describeViewports.test.js
  describeViewports
    ✕ should describe viewport (5 ms)

  ● describeViewports › should describe viewport

    TypeError: expect(...).toHaveBeenCalledOnce is not a function

      19 |     describeViewports('viewport', ['tiny', 'large'], () => {});
      20 | 
    > 21 |     expect(global.describe).toHaveBeenCalledOnce();
         |                             ^
      22 |   });
      23 | });
      24 | 

      at Object.<anonymous> (packages/terra-functional-testing/tests/jest/commands/utils/describeViewports.test.js:21:29)
```